### PR TITLE
Update fr.js

### PR DESCRIPTION
**Pull request by rezeau**
Updated French translation: translated untranslated strings & fixed errors.

### CODE REVIEW

File: language/fr.js [hunk 1]
```diff
@@ -1,57 +1,56 @@
...
-  onlyNumbers: 'La :property ne peut contenir que des nombres.',
-  exceedsMax: 'La :property dépasse le maximum de :max.',
-  belowMin: 'La :property dépasse le minimum de :min.',
+  requiredProperty: 'Le champ :property est obligatoire et ne peut être vide.',
+  onlyNumbers: 'Le champ :property ne peut contenir que des nombres.',
+  exceedsMax: 'La valeur du champ :property est supérieure au maximum de :max.',
+  belowMin: 'La valeur du champ :property est inférieure au minimum de :min.',
   outOfStep: 'La :property ne peut être effectuée que par étape  :step.',
-  add: 'Add',
   addFile: 'Ajouter un fichier',
```
Comment (timothyylim) [inline language/fr.js]: Are you missing 'add:' in the new translation?

**Comment by rezeau** [comment]
@timothyylim My mistake, I built my translation from the current version of the fr file on my platform and I notice that the add string was added more recently, sorry.
French trans should be add: 'Ajouter'
Can you please fix this for me or should I pull a new request?

File: language/fr.js (hunk 1 above)
Comment (timothyylim) [inline language/fr.js]: @rezeau, you could edit the file and continue with the pull request :)

**Comment by icc** [comment]
Thank you! Works good.

**Pull request merged**